### PR TITLE
Improve the style for devise views

### LINKED ISSUE #40: Improve the style for devise views
# REQUIREMENTS
Devise views need to have a more user-friendly design

# Acceptance Criteria
List all required acceptance criteria
- [ Add styles for devise views ] Criteria 1
- [ Update the links and labels languages to spanish] Criteria 2

### PR DESCRIPTION
**Pull request by Frcomparan**
# Description
What did you implemented/Why did you implemented this:
What?
 - Added styles for devise forms
 - Added translation for locales
 - Added config to implement emails in production

Why?
 - To give a better design to the page
 - To show all the information in a single language
 - To fix a problem with emails in production

  ## Screenshots 
### Sign in
image

### Recover password
image

### Edit user
image

### CODE REVIEW

**Review by MrCano369x — DISMISSED** [review]
lgtm

File: app/views/devise/shared/_links.html.erb [hunk 1]
```diff
@@ -1,25 +1,25 @@
...
 <% end %>
 
 <%- if devise_mapping.registerable? && controller_name != 'registrations' %>
-  <%= link_to "Sign up", new_registration_path(resource_name) %><br />
+  <%= link_to "Registrate", new_registration_path(resource_name) %><br />
 <% end %>
 
 <%- if devise_mapping.recoverable? && controller_name != 'passwords' && controller_name != 'registrations' %>
-  <%= link_to "Forgot your password?", new_password_path(resource_name) %><br />
+  <%= link_to "¿Olvidate tu contraseña?", new_password_path(resource_name) %><br />
```
Comment (dannharo) [inline app/views/devise/shared/_links.html.erb]: I think there is a typo here

Comment (Frcomparan) [reply app/views/devise/shared/_links.html.erb]: Yes it was. Fixed typo. Great view.

**Review by YilKanda — APPROVED** [review]
LGTM!

**Review by JuanequeX — APPROVED** [review]
LGTM

**Pull request merged**